src/functions-reference/positive_lower-bounded_distributions.qmd

```diff
@@ -215,6 +218,46 @@ and one representation that converges fast for large reaction-time values ("larg
 \end{equation*}
 see [@Feller1968], [@NavarroFuss2009].
 
+### Cumulative distribution function
+
+For the cumulative distribution function there also exist two expressions depending on the reaction time.
+
+Let $\alpha$, $\tau$, $\beta$, $\delta$, $s_{\delta}$, $s_{\beta}$, $s_{\tau}$ and $y$ be as above.
```

Comment:
Can you reduce lines to 80-ish characters?  It becomes very hard to review diffs with long lines.